release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -17,7 +17,7 @@ The Release Team Lead should be an arbiter of decisions, and not the primary dec
   - You must be a member in good standing, and part of the Kubernetes org
   - You must be active in at least one Kubernetes SIG
   - You must be a member of the kubernetes-dev mailing list
-- Several release team members are in the Pacific Time Zone partially due to infrastructure access limitations at Google, therefore, the release lead must be comfortable scheduling meetings and events compatible with PT working hours
+- Historically release team members have been focused close to the Pacific Time Zone partially due to infrastructure access limitations at Google.  These limitations are most entirely legacy at this point.  Therefore, the release lead should be comfortable scheduling meetings and events compatible with global working hours and coordinating a global set of team leads and shadows.  This may mean having the traditional 10am Pacific release team meeting, but also scheduling alternate time meetings and varying the final weeks' burndown meeting time slots to best accommodate the individuals (both release team and broader SIG representation on the issues of the day) whose presence and information is needed.  Sacrifice may be necessary at times and the release lead should endeaver to spread this so as not to focus any inconvenience on specific individuals or specific geographies, while also making sure the project is correctly moving forward.
```

Comment:
s/"coordinating a global set of team leads and shadows"/"coordinating **with** a global set of team leads and shadows"

Reply:
Can we also include some text on the release lead taking global holidays into account when schedulng meetings and release calendar itself.